.github/workflows/release-preparations.yml

```diff
@@ -0,0 +1,54 @@
+name: Release preparations
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        default: patch
+        description: SemVer release level
+        options:
+          - major
+          - minor
+          - patch
+        required: true
+        type: choice
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  aeruginous:
+    if: |
+      contains(fromJson('["major", "minor", "patch"]'), inputs.release) &&
+      github.repository == 'sourcefrog/cargo-mutants'
+    runs-on: ubuntu-latest
+    steps:
+      - run: rustup update
+      - uses: actions/checkout@v4
+      - uses: fregante/setup-git-user@v2
+
+      - uses: baptiste0928/cargo-install@v2
+        with:
+          crate: aeruginous
```

Comment:
Calling Aeruginous requires installing it, at first.  This Action will also cache the binary and dependencies for future usage to enhance the performance of this workflow.